Extensions/TextObject/Extension.cpp

```diff
@@ -200,6 +200,70 @@ void DeclareTextObjectExtension(gd::PlatformExtension& extension) {
       .SetFunctionName("SetColor")
       .SetIncludeFile("TextObject/TextObject.h");
 
+  obj.AddAction(
+         "ChangeGradient",
+         _("Gradient"),
+         _("Change the gradient of the text."),
+         _("Change gradient of _PARAM0_ to colors _PARAM1_ _PARAM2_ _PARAM3_ _PARAM4_ type _PARAM5_"),
+         _("Effects"),
+         "res/actions/textGradient24.png",
+         "res/actions/textGradient.png")
+
+      .AddParameter("object", _("Object"), "Text")
+      .AddParameter("color", _("First Color"))
+      .AddParameter("color", _("Second Color"))
+      .AddParameter("color", _("Third Color"))
+      .AddParameter("color", _("Fourth Color"))
+      .AddParameter("stringWithSelector", _("Gradient type"), "[\"LINEAR_VERTICAL\", \"LINEAR_HORIZONTAL\"]", false)
+      .SetFunctionName("SetGradient")
+      .SetIncludeFile("TextObject/TextObject.h");
+
+  obj.AddAction(
+         "ChangeOutline",
+         _("Outline"),
+         _("Change the outline of the text. A thickness of 0 disables the outline."),
+         _("Change outline of _PARAM0_ to color _PARAM1_ with thickness _PARAM2_"),
+         _("Effects"),
+         "res/actions/textOutline24.png",
+         "res/actions/textOutline.png")
+
+      .AddParameter("object", _("Object"), "Text")
+      .AddParameter("color", _("Color"))
+      .AddParameter("expression", _("Thickness"))
+      .SetFunctionName("SetOutline")
+      .SetIncludeFile("TextObject/TextObject.h");
+
+  obj.AddAction(
+         "ChangeShadow",
```

Comment:
Can you call this `SetShadow`, and other actions `SetXXX` instead of `ChangeXXX`?

I know there is no such convention in current GDevelop actions, but going forward I'd like to have, if `XXX` is the condition name, `SetXXX` for the action and `XXX` for the expression.